tests/util.test.ts

```diff
@@ -14,15 +14,16 @@
  * limitations under the License.
  */
 
-import { expect } from 'chai';
 import 'mocha';
-import { GetDestinationFromPath } from '../src/util';
+import { expect } from 'chai';
+
+import { getDestinationFromPath } from '../src/util';
```

Comment:
I like this goimport like grouping. Did you do this manually or is this a lint setting?

Reply:
Manually unfortunately